tests/macros/fixtures/defaultapisidebar/commonl10n.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Just a small point that it looks like only 7 of the 12 keys are actually used by the `DefaultAPISidebar` macro (`Methods`, `Properties`, `Events`, `Interfaces`, `Guides`, `[Translate]`, and `TranslationCTA`), so this file could be smaller if desired.